libafl_frida/src/coverage_rt.rs

```diff
@@ -158,9 +158,9 @@ impl CoverageRuntime {
         let bitflip = 0x1cad21f72c81017c ^ 0xdb979082e96dd4de;
         let mut h64 = tmp ^ bitflip;
```

Comment:
Is this a hashing function we would want to move to bolts or targets, btw?

Reply:
bolts/rands.rs maybe?

Reply:
It's not an RNG though, so no

Reply:
I see.
then I can put it in libafl_frida/src/utils.rs, and mark it as inline function

Reply:
Is it only useful for Frida?

Reply:
ok then probably I'll make a new file bolts/hasher.rs